wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -864,6 +864,7 @@ private function execute_create_table() {
 		$table = $this->parse_create_table();
 
 		$definitions = array();
+		$on_updates  = array();
```

Comment:
What does this array do?

Should it be used in the loop on line 926?

Reply:
@bgrgicak Ahh, good catch! Originally, I was looping the array, but then I realized I can simply loop the fields, and seems like I somehow forgot to completely remove it. Which variant do you think is better? With or without the `$on_updates` array?

Reply:
With `on_update` seems better to me.

Reply:
@bgrgicak Thanks! Addressed in https://github.com/WordPress/sqlite-database-integration/pull/150/commits/50279c4c126bb091ae62abc4c44ead9136a2bc88.